api/wireformat/entities.go

```diff
@@ -51,15 +51,15 @@ type PlanActive struct {
 // Plan structure is used as a wire format to store information on ISV-created
 // rating plan and charm URLs for which the plan is valid.
 type Plan struct {
-	Id              string      `json:"id"`         // Full id of the plan format
-	URL             string      `json:"url"`        // Name of the rating plan
-	Definition      string      `json:"plan"`       // The rating plan source
-	CreatedOn       string      `json:"created-on"` // When the plan was created - RFC3339 encoded timestamp
-	PlanDescription string      `json:"description"`
-	PlanPrice       string      `json:"price"`
-	Released        bool        `json:"released"`
-	EffectiveTime   *time.Time  `json:"effective-time,omitempty"`
-	Model           interface{} `json:"model,omitempty"` // The rating plan model
+	Id              string      `json:"id" yaml:"id"`                 // Full id of the plan format
+	URL             string      `json:"url" yaml:"url"`               // Name of the rating plan
```

Comment:
URL is part of the Id, right? If so, do we need to output it?

Reply:
this was not added in this PR..